include/swift/AST/Decl.h

```diff
@@ -1162,15 +1166,61 @@ class ImportDecl final : public Decl,
     return static_cast<ImportKind>(Bits.ImportDecl.ImportKind);
   }
 
+  /// Retrieves the import path as written in the source code.
+  /// 
+  /// \returns An \c ImportPath corresponding to this import decl. If module
+  ///          aliasing was used, this will contain the aliased name of the module;
+  ///          for instance, if you wrote 'import Foo' but passed
+  ///          '-module-alias Foo=Bar', this import path will include 'Foo'. This
+  ///          return value is always owned by the AST context, so it can be
+  ///          persisted.
   ImportPath getImportPath() const {
-    return ImportPath({ getTrailingObjects<ImportPath::Element>(),
-                        static_cast<size_t>(Bits.ImportDecl.NumPathElements) });
+     return ImportPath({ getTrailingObjects<ImportPath::Element>(),
+                         static_cast<size_t>(Bits.ImportDecl.NumPathElements) });
```

Comment:
Nit: The indentation added here is incorrect. (This is probably an artifact of how GitHub handles copy-and-paste.)